node/src/main.rs

```diff
@@ -318,6 +334,23 @@ fn async_main() -> impl Future<Item = (), Error = ()> + Send + 'static {
 
     debug!(logger, "Setting up Sentry");
 
+    // Obtain DISABLE_BLOCK_INGESTOR setting
+    let disable_block_ingestor: bool = matches
+        .value_of("disable-block-ingestor")
+        .unwrap()
+        .parse()
+        .expect("invalid --disable-block-ingestor/DISABLE_BLOCK_INGESTOR value");
+
+    // Obtain STORE_CONNECTION_POOL_SIZE setting
+    let store_conn_pool_size = matches
+        .value_of("store-connection-pool-size")
+        .unwrap()
+        .parse()
+        .expect("invalid --store-connection-pool-size/STORE_CONNECTION_POOL_SIZE value");
+
+    // Minimum of two connections needed for the pool in order for the Store to bootstrap
+    assert!(store_conn_pool_size > 1);
```

Comment:
This will generate a pretty ugly error. Also: Are assertions still being executed in production builds?

How about this?
```rust
if (store_conn_pool_size < 1) {
  panic!("--store-connection-pool-size/STORE_CONNECTION_POOL_SIZE must be > 1");
}
```